lib/crawler.js

```diff
@@ -1380,21 +1474,27 @@ Crawler.prototype.handleResponse = function(queueItem, response, timeCommenced)
         queueItem.status = "redirected";
 
         // Parse the redirect URL ready for adding to the queue...
-        referrerQueueItem = crawler.processURL(response.headers.location, queueItem);
+        redirectQueueItem = crawler.processURL(response.headers.location, queueItem);
 
-        // Emit redirect event
-        crawler.emit("fetchredirect", queueItem, referrerQueueItem, response);
```

Comment:
This used to be plain wrong. `referrerQueueItem` was always the queue item for the next request, so I fixed the wording here